docs/modules/ROOT/pages/explanation/organizations-and-rbac.adoc

```diff
@@ -6,11 +6,17 @@ The organization is also reflected as a `Group` object in the {zone} and is used
 
 == Default RBAC rules
 
-A `RoleBinding` with the name `admin` is created in the `Namespace` which grants the creator the `ClusterRole` called `admin`.
-Another `RoleBinding` with the name `organization-admin` is created in the `Namespace` which also grants the organization `admin` cluster role.
-The user is free to modify or delete either.
+On `Namespace` creation, a number of RBAC rules are created.
+
+* A `RoleBinding` called `admin` which grants the creator of the `Namespace` the `ClusterRole` called `admin`.
+This roles allows the creator to create, get, or modify nearly every resource in the `Namespace`.
+* A `RoleBinding` called `organization-admin` which also grants the organization the `admin` role.
+* A `RoleBinding` and `Role` called `namespace-owner` which gives the organization the permission to edit the created `Namespace`.
+
+The user is free to modify or delete any of the `Roles` or `RoleBindings` and handle access control themselves.
 
 IMPORTANT: Be sure to not lock yourself out!
+Nothing will stop you from removing access to created projects.
```

Comment:
Does this render correctly?
Shouldn't it be enclosed?
```suggestion
[IMPORTANT]
====
Be sure to not lock yourself out!
Nothing will stop you from removing access to created projects.
====
```

Reply:
This renders correctly, but you're right, enclosing it is nicer